app/views/custom/layouts/_footer.html.erb

```diff
@@ -0,0 +1,38 @@
+<footer>
+  <div class="row">
+    <div class="small-12 large-4 column">
+      <h1 class="logo">
+        <%= link_to "https://pro-ethics.eu", title: t("layouts.footer.framework") do %>
+          <%= image_tag "ProEthics.svg", width: "250px" %>
```

Comment:
I think we need an `alt` attribute here; otherwise, it's invalid HTML and an accessibility issue.